packages/react-scripts/template/README.md

```diff
@@ -205,6 +205,34 @@ class Button extends Component {
 export default Button; // Don’t forget to use export default!
 ```
 
+To import your own modules into other files, you can use relative paths by default. For example:
+```js
+import Banana from '../../Banana';
+```
+You can also enable absolute paths by adding a NODE_PATH environment variable. This is a bit of a stop-gap measure for now. Here is an example absolute import and the commands you would need to run for it to work:
```

Comment:
Let’s expand “This is a bit of a stop-gap measure for now.” into “We don’t recommend this at the moment, and we encourage you to use relative paths for your projects if you can. However, this can be used as a stop-gap measure if you’re porting a large project to Create React App. In the future, we intend to offer a better way of handling absolute imports.”